.github/workflows/integration.yml

```diff
@@ -0,0 +1,75 @@
+name: Integration testing
+env:
+  PROJECT_ROOT: /root/go/src/github.com/infrawatch/sg-core
+
+  QDR_IMAGE: quay.io/interconnectedcloud/qdrouterd:1.15.0
+  QDR_VOLUME: "--volume=${{ github.workspace }}/ci/service_configs/qdr:/etc/qpid-dispatch:ro"
+  QDR_PORT: "-p 5666:5666"
+
+  LOKI_IMAGE: grafana/loki:2.1.0
+  LOKI_VOLUME: "--volume=${{ github.workspace }}/ci/service_configs/loki:/etc/loki:ro"
+  LOKI_PORT: "-p 3100:3100"
+
+  TEST_IMAGE: quay.io/centos/centos:stream8
+
+on: push
+
+jobs:
+  logging:
+    name: "[logging] handler: logs; application: elasticsearch, loki"
+    runs-on: ubuntu-20.04
+    services:
+      elastic:
+        image: docker.elastic.co/elasticsearch/elasticsearch:7.10.2
```

Comment:
Move this in to an ENV like other images.

Reply:
Ah I see, ok will do.